pulp_2to3_migration/app/pre_migration.py

```diff
@@ -286,14 +286,19 @@
                          all are migrated.
     """
     mongo_importer_q = mongo_Q(repo_id=repo.pulp2_repo_id)
+
+    # importers with empty config are not needed - nothing to migrate
+    mongo_importer_q &= mongo_Q(config__ne={})
```

Comment:
it might worth adding a check if feed is null, wdyt? remotes in pulp3 are tightly coupled with the feed